docs/pages/access-controls/reference.mdx

```diff
@@ -172,6 +174,17 @@ The table below documents the behavior of each option if multiple roles are assi
 | `max_connections` | Limit on how many active SSH sessions can be started via Teleport | |
 | `max_sessions` | Total number of session channels which can be established across a single SSH connection via Teleport | |
 
+## Preset roles
+
+Teleport provides several pre-defined roles out-of-the-box:
+
+| Role | Description |
+| --- | --- |
+| `admin` | Full cluster administrators. |
```

Comment:
`Full cluster administrators.` feels a little fragmented. How about `Allows complete control over cluster setting, resources and event.`